remediation/code-samples/02-countValue.js

```diff
@@ -14,6 +14,8 @@
  * @param {object} - A value to count
  * @returns {number} The number of times the value appears in the array
  */
+
+ // PLEASE REVIEW
 function countValue(array, value) {
   for(i=0;i<array.length;i++)
```

Comment:
You have the right idea here. We DO need to loop through all of the elements in the array and check to see if they are equal to value.

But the way you are currently approaching this problem causes us to skip checking some values! For example...
`countValue([1,2,3,3,3,4],3)
`
1. [1,2,3,3,3,4]      i=0, 1!=3 so we remove 1 and i++
2. [2,3,3,3,4]         i=1, 3==3 so we continue and i++
3. [2,3,3,3,4]         i=2, 3==3 so we remove 1 and i++
4. [2,3,3,3,4]         i=3, 3==3 so we remove 1 and i++
5. [2,3,3,3,4]         i=4, 4!=3 so we remove 4 and i++
5. [2, 3,3,3]           i=5, loop ends

By removing an element AND increasing `i`, we effectively jump two indexes forward! 
Instead of modifying the array and removing elements as you go, is there another way to keep track of how many times you found `value`? Maybe with another variable...?